de-webapp/src/main/webapp/WEB-INF/jsp/bootstrapError.jsp

```diff
@@ -173,10 +193,38 @@ a:hover {
 
     <hr/>
 
-    <div style="float: left"><a href="${login_url}">Go back to ${app_name}</a></div>
     <div style="float: right"><a href="http://www.cyverse.org">CyVerse Home Page</a></div>
+    <div><a href="${login_url}">Go back to ${app_name}</a></div>
+    <div><a href="${logout_url}">Log Out</a></div>
 
 </div>
 
 </body>
 </html>
+
+<script type="text/javascript">
+    <%--If the user refreshes their browser, redirect to /de --%>
+    if (sessionStorage.getItem("is_reloaded")) {
+        window.location.replace('${login_url}');
+        sessionStorage.removeItem("is_reloaded");
+    } else {
+        sessionStorage.setItem("is_reloaded", 1);
+    }
+
+    var acc = document.getElementById("errorBtn");
+
+    acc.addEventListener("click", function() {
```

Comment:
This looks good for this PR, but I just want to point out that we should be able to use React components here, if we use the following JSP include that our main `de.jsp` page uses:
https://github.com/cyverse-de/ui/blob/7bc4e878f27fd90295a0ef0aad5f88aee3aae52e/de-webapp/src/main/webapp/WEB-INF/jsp/de.jsp#L68

Then we could use material-ui components (and our custom CyVerse theme) and display a custom component with the error message here.

For example:
```javascript
        ReactDOM.render(
            React.createElement(
                CyVerseReactComponents.util.ErrorExpansionPanel,
                {
                    errMsg: "${error_message}",
                    username: "${username}",
                    userAgent: "${user_agent}",
                    date: "${date}",
                    host: "${request_url}"
                }
            ),
            document.getElementById(acc)
        );
```
By the way, since our `CyVerseReactComponents` javascript library is a global object on the page, it can be examined from the browser's console when the DE is loaded.

Reply:
Oh, neat - I'll give that a go!  I was thinking material would be better, but wasn't sure how to put it directly into the JSP.